src/sentry/web/frontend/js_sdk_loader.py

```diff
@@ -34,23 +32,47 @@ def get(self, request, public_key, minified):
         except TypeError:
             sdk_url = ''  # It fails if it cannot inject the version in the string
 
+        return ({
+            'config': config.get_project_key_config(key),
+            'jsSdkUrl': sdk_url,
+            'publicKey': key.public_key
+        }, sdk_version, sdk_url)
+
+    def get(self, request, public_key, minified):
+        """Returns a js file that can be integrated into a website"""
+        start_time = time.time()
+        key = None
+
+        try:
+            key = ProjectKey.objects.get(
+                public_key=public_key
+            )
+        except ProjectKey.DoesNotExist:
+            pass
+
+        context, sdk_version, sdk_url = self._get_context(key)
+
+        instance = "default"
         if not sdk_url:
+            instance = "noop"
             tmpl = 'sentry/js-sdk-loader-noop.js.tmpl'
         elif minified is not None:
+            instance = "minified"
             tmpl = 'sentry/js-sdk-loader.min.js.tmpl'
         else:
             tmpl = 'sentry/js-sdk-loader.js.tmpl'
 
-        context = {
-            'config': config.get_project_key_config(key),
-            'jsSdkUrl': sdk_url,
-            'publicKey': public_key
-        }
+        metrics.incr('js-sdk-loader.rendered', instance=instance)
 
         response = render_to_response(tmpl, context, content_type="text/javascript")
 
```

Comment:
Can we just add a test to assert we’re setting this so we don’t accidentally break it in the future?